tests/test_kernel_module.py

```diff
@@ -48,6 +48,31 @@ def create_kernel_test(containerfile: str) -> ParameterSet:
 )
 
 
+KMP_CONTAINER = create_kernel_test(
+    r"""WORKDIR /src
+RUN set -euxo pipefail; \
+    curl -sf https://src.opensuse.org/pool/vhba-kmp/archive/f7b947967abfd854991eed3a50a333f2efa4c47932e3d7d2fc1d07c6afdb79bd.tar.gz | tar -xzf - ; \
```

Comment:
What's this specific commit?

Reply:
its just master of today. wanted something predictable for the source curl download afterwards.

Reply:
```suggestion
    # latest commit in the factory branch on https://src.opensuse.org/pool/vhba-kmp as of 2024-12-03
    curl -sf https://src.opensuse.org/pool/vhba-kmp/archive/f7b947967abfd854991eed3a50a333f2efa4c47932e3d7d2fc1d07c6afdb79bd.tar.gz | tar -xzf - ; \
```

Reply:
there's a comment at the beginning of the section now.